### ISSUE TITLE
[MultiSelect Field] When a multiselect field is a reference, it could create 400 errors on POST request

### ISSUE THREAD

**Issue by mdartic** [locokit/locokit#33]
On a use case where we define a MultiSelect field as a reference for the table,
this lead the compute of the `text` property to an error, because we explode the varchar 255 in database.

Maybe we need to forbid the fact that a MultiSelect field could be a reference ? (this doesn't make sense I think)

Any idea @DarkVisenya @jcougnaud ?

**Comment by jcougnaud** [comment]
Yes, I'm not sure but maybe none of the column types with limited choices (Boolean, SingleSelect, MultiSelect, RelationBetweenTables, LookUpColumn, ... ?) should be chosen as references since the goal is to uniquely identify a row.
